src/document.rs

```diff
@@ -140,6 +147,13 @@ fn document_attributes(
     ];
     let mut attributes: Object = attributes.into_iter().collect();
 
+    if let Some(ref tags) = front.tags {
+        attributes.insert(
+            "tags".into(),
+            Value::scalar(itertools::join(tags.iter().map(slug::slugify), ", ")),
```

Comment:
This is for the `page.tags` variable, correct?

1. Why are we slugifying it?  Should that instead be something in the frontmatter builder?
2. By us doing a join on this, we are preventing people from creating links to the tags, preventing custom formatting, etc.  The RFC doesn't explicitly list the data type but it does say you can do `{% for tag in page.tags %}` which implies an `Array` of strings.

Reply:
my bad, copy pasted from permalink (haste haste… bad habits)